main/design_documents/pacbti-tfm.rst

```diff
@@ -67,6 +67,26 @@ Example:
   #endif
   }
 
+
+Example (AArch64):
+--------
+
+::
+
+  #if __ARM_FEATURE_PAC_DEFAULT
+    // Authenticated return required.
+  # if __ARM_FEATURE_PAUTH
+    // Have PAUTH instructions
+    asm ("retaa");
+  # else
+    // Use authentication from NOP space.
+    asm ("autiasp;ret");
+  # endif
+  #else
+    // No authentication required.
+    asm ("ret");
+  #endif
+
 A real-world use-case of these macros is in the stack-unwinding runtime for
 exceptions where the runtime unwinding is potentially better with ``AUTG``
 rather than using ``AUT`` which forces register pressure.
```

Comment:
You can't 'force' register pressure.  You can, however, increase or decrease it.  I think it would be better to change this sentence to:
A more practical example of these macros is in the stack-unwinding runtime for exceptions.  In that case the runtime unwinding is improved by using ``AUTG`` rather than ``AUT`` because it frees up an additional register.

Reply:
Okay, this text was already present. @fpetrogalli to improve this do you prefer to have this change:
- in the same commit 
- in a different commit in the same PR
- in a separate PR?
I'd suggest the second option.
Thanks
  Andrea

Reply:
@AndreaCorallo  - new commits on top of current, in the same PR (just push your branch). New commits make review easier to track. Once the PR is approved and ready, I'll squash them in a single commit.